lib/sendgrid_actionmailer.rb

```diff
@@ -163,7 +163,18 @@ def add_content(sendgrid_mail, mail)
     end
 
     def json_parse(text, symbolize=true)
```

Comment:
I wonder if this method should be renamed. It seems to be more focused on converting values to a Hash rather than parsing JSON, the actual JSON parsing is really an implementation detail - a detail which isn't used in all cases now.